src/pages/workspaces/workspace/workflows-cbas/SubmissionHistory.js

```diff
@@ -0,0 +1,15 @@
+import { h2 } from 'react-hyperscript-helpers';
+import { wrapWorkflowsPage } from 'src/pages/workspaces/workspace/workflows-cbas/WorkflowsContainer';
+
+export const SubmissionHistory = wrapWorkflowsPage({ name: 'SubmissionHistory' })((props, _ref) => {
+  return h2(['Submission History page']);
+});
+
+export const navPaths = [
+  {
+    name: 'workspace-workflows-cbas-submission-history',
+    path: '/workspaces/:namespace/:name/workflows-cbas/submission-history',
+    component: SubmissionHistory,
+    title: ({ name }) => `${name} - Workflows`,
```

Comment:
Similar comment here. Should this have something else apart from `Workflows` in title?